themes/rainbowstone/webapp/rainbowstone/style.css

```diff
@@ -1927,4 +1927,12 @@ html > /**/ body .jstree-default a {
 /* The custom CSS to show mouse pointer on links using featherlight plugin  */
 a[data-featherlight] {
     cursor: pointer;
-}
\ No newline at end of file
+}
+/*The custom css for Trumbowyg*/
+.trumbowyg-button-pane {
```

Comment:
There seems an indentation issue in css in both the files. Could you please fix that?
Rest looks good to me. Thanks for your updates!